### PR TITLE
change top-level for datasource properties; clean up data source beans a bit

### PR DESCRIPTION
**Pull request by malaffoon**
Two sets of changes:
* change top-level for datasource properties from `spring` to `datasources`; make the datasource property names consistent with RDW_Reporting, e.g. `warehouse_rw`
* clean up data source beans a bit: remove unnecessary bean names, make names consistent

### CODE REVIEW

File: common/src/test/java/org/opentestsystem/rdw/ingest/common/repository/CommonTestConfiguration.java [hunk 1]
```diff
@@ -2,14 +2,27 @@
...
 @Import({ YamlPropertiesConfigurator.class })
 public class CommonTestConfiguration {
+
+    @Bean
+    @ConfigurationProperties(prefix = "datasources.warehouse_rw")
+    public DataSource dataSource() {
+        return DataSourceBuilder
+                .create()
+                .build();
+    }
```
Comment (malaffoon) [inline common/src/test/java/org/opentestsystem/rdw/ingest/common/repository/CommonTestConfiguration.java]: Need this to stop Spring's autoconfiguration

File: migrate-olap/src/test/java/org/opentestsystem/rdw/ingest/migrate/olap/MigrateOlapReportingJobRST.java [hunk 2]
```diff
@@ -54,13 +54,13 @@ public void cleanUp() {
     }
 
     @SqlGroup({
-            @Sql(executionPhase = BEFORE_TEST_METHOD, scripts = {"classpath:WarehouseImportSetup.sql"}, config = @SqlConfig(dataSource = "warehouseDatasource")),
-            @Sql(executionPhase = BEFORE_TEST_METHOD, scripts = {"classpath:WarehouseCodesForDeleteSetup.sql"}, config = @SqlConfig(dataSource = "warehouseDatasource")),
-            @Sql(executionPhase = BEFORE_TEST_METHOD, scripts = {"classpath:OlapCodesPreload.sql"}, config = @SqlConfig(dataSource = "olapDatasource")),
+            @Sql(executionPhase = BEFORE_TEST_METHOD, scripts = {"classpath:WarehouseImportSetup.sql"}, config = @SqlConfig(dataSource = "warehouseDataSource")),
+            @Sql(executionPhase = BEFORE_TEST_METHOD, scripts = {"classpath:WarehouseCodesForDeleteSetup.sql"}, config = @SqlConfig(dataSource = "warehouseDataSource")),
+            @Sql(executionPhase = BEFORE_TEST_METHOD, scripts = {"classpath:OlapCodesPreload.sql"}, config = @SqlConfig(dataSource = "olapDataSource")),
```
Comment (malaffoon) [inline migrate-olap/src/test/java/org/opentestsystem/rdw/ingest/migrate/olap/MigrateOlapReportingJobRST.java]: I *almost* regret fixing the datasource names

File: common/src/test/java/org/opentestsystem/rdw/ingest/common/repository/CommonTestConfiguration.java (hunk 1 above)
Comment (ftdholbrook) [reply common/src/test/java/org/opentestsystem/rdw/ingest/common/repository/CommonTestConfiguration.java]: an alternative 
`@EnableAutoConfiguration(exclude = {DataSourceAutoConfiguration.class, DataSourceTransactionManagerAutoConfiguration.class, HibernateJpaAutoConfiguration.class})`

**Comment by ftdholbrook** [comment]
lots of touch points.  :-) looks good to me.

File: common/src/test/java/org/opentestsystem/rdw/ingest/common/repository/CommonTestConfiguration.java (hunk 1 above)
Comment (malaffoon) [reply common/src/test/java/org/opentestsystem/rdw/ingest/common/repository/CommonTestConfiguration.java]: I was leaving that for the next step. Even with that i'd have to have this to get the data source for the test.

**Pull request merged**